### DISCUSSION TITLE
Alpha Play Test (B.o.S. Version 0.003) Looking For Feedback.

### DISCUSSION THREAD

**Discussion by WithinAmnesia** [flareteam/flare-game#935] [Show and tell]
Download: https://mega.nz/file/rdgWxRhZ#n9pyTlI_vgaY8cnSRlLL9GnXixXNH_henX3kNhH-gVg
Alpha Play Test.
![Lots of Adventure!](https://user-images.githubusercontent.com/86871191/221393086-2fe83bf5-9c23-4e48-9e88-2918b1b828a2.png)
![I made most of the icons a bit sharper](https://user-images.githubusercontent.com/86871191/221393094-0dc258cf-dd90-45c1-9627-63e011c68280.png)
![It's looking fun! Alpha Play Demo](https://user-images.githubusercontent.com/86871191/221393098-f3e17ec0-bfbd-43e0-b373-f92a5bbb8e1f.png)
![The first iteration of the full scale functional interface!](https://user-images.githubusercontent.com/86871191/221393106-c093f64d-a0c3-46cb-9857-b0ef54f36d00.png)
Previous Version: https://github.com/flareteam/flare-game/discussions/930

**Comment by Paul-Wortmann** [comment]
As I run on Linux, I built the engine from the latest source. I couldn't use the included Windows binary.
While I can get into the game, albeit with difficulty, I am greeted with a horribly scaled UI and a black play area.
Your screen shots above look interesting, and I'm looking forward to a future build with a more functional UI.
BTW, I used a resolution of 1920x1080
![Screenshot from 2023-05-07 07-42-58](https://user-images.githubusercontent.com/830729/236649552-8104c152-f3a1-440c-8cd7-57dfc059c2f0.png)
